static/app/components/onboarding/productSelection.tsx

```diff
@@ -19,9 +21,15 @@ export enum PRODUCT {
 
 type Props = {
   defaultSelectedProducts?: PRODUCT[];
+  lazyLoader?: boolean;
+  skipLazyLoader?: () => void;
```

Comment:
what if we only use the `skipLazyLoader` for the check? if it is provided we render everything we are currently rendering for `lazyLoader` otherwise the default version

Reply:
hmm, not sure, feels a bit less explicit to me, but happy to do however you feel works best!

Reply:
we can do this in  a follow-up 😉